### PR TITLE
Updating STI with current v0.3 release

### PR DESCRIPTION
**Pull request by soltysh**
@bparees PTAL

### CODE REVIEW

**Comment by soltysh** [comment]
[test]

File: images/builder/docker/sti-image-builder/Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ FROM centos:centos7
 
 RUN yum install -y --enablerepo=centosplus epel-release gettext tar automake make git docker
 
-ADD https://github.com/openshift/source-to-image/releases/download/v0.2/source-to-image-v0.1-80-gad5adc05431168-ad5adc0-linux-amd64.tar.gz /usr/bin/sti.tar.gz
+ADD https://github.com/openshift/source-to-image/releases/download/v0.3/source-to-image-v0.3-957c66b-darwin-amd64.tar.gz /usr/bin/sti.tar.gz
```
Comment (bparees) [inline images/builder/docker/sti-image-builder/Dockerfile]: shouldn't this be linux-amd?

Comment (soltysh) [inline images/builder/docker/sti-image-builder/Dockerfile]: Yes it should.

**Comment by bparees** [comment]
lgtm. [merge]

**Comment by soltysh** [comment]
re-[test]

**Pull request merged**